bumble/device.py

```diff
@@ -433,6 +450,12 @@ def send_l2cap_pdu(self, cid, pdu):
     def create_l2cap_connector(self, psm):
         return self.device.create_l2cap_connector(self, psm)
 
+    def while_alive(self, coro_or_future):
+        future = self.device.while_alive(coro_or_future)
+        listener = self.on('disconnection', lambda _: future.set_exception(ConnectionDiedError))
+        future.add_done_callback(lambda _: self.remove_listener('disconnection', listener))
+        return future
```

Comment:
Though I can guess its usage but it will be better to have some description here.